static/scss/audio-player.scss

```diff
@@ -0,0 +1,135 @@
+.audio-player-container-outer {
+  background-color: #a01e35;
+  position: fixed;
+  left: 0;
+  bottom: 0;
+  width: 100vw;
+}
+
+.audio-player-container-inner {
+  position: relative;
+  display: flex;
+  flex: 1;
+  width: 100%;
+  padding: 8px 12px;
+}
+
+.audio-player-titles {
+  padding-left: 12px;
+  padding-right: 12px;
+  border-right: 1px solid rgba(0, 0, 0, 0.1);
+}
+
+.audio-player-podcast-name {
+  font-weight: 400;
+  font-size: 16px;
+  color: #8e8e8e;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  max-height: 25px;
+}
+
+.audio-player-podcast-title {
+  font-weight: 500;
+  font-size: 18px;
+  color: white;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  max-height: 55px;
+  line-height: 130%;
+}
+
+.audio-player-controls {
+  color: white;
+  display: flex;
+  flex-direction: row;
+  flex: 1;
+  justify-content: center;
+  align-items: center;
+}
+
+.audio-player-button {
+  height: 40px;
+  display: flex;
+  align-items: center;
+  padding-left: 12px;
+  padding-right: 12px;
+}
+
+.audio-player-button span,
+progress {
+  cursor: pointer;
+}
+
+.amplitude-play-pause span {
+  font-size: 3em;
+}
+
+.amplitude-play-pause span:before {
+  content: "play_circle_outline";
+}
+
+.amplitude-play-pause.amplitude-playing span:before {
+  content: "pause_circle_outline";
+}
+
+.amplitude-play-pause.amplitude-paused span:before {
+  content: "play_circle_outline";
+}
+
+.audio-player-button:not(.amplitude-play-pause) span {
+  font-size: 2em;
+}
+
+.audio-player-time-text-left {
+  text-align: right;
+  padding-right: 12px;
+}
+
+// disabling no-vendor-prefixes here because autoprefixer doesn't support html5 progress bars
+// sass-lint:disable no-vendor-prefixes
+.audio-player-progress {
+  background-color: #ebc1b6;
+  color: white;
+  -webkit-appearance: none;
+}
+
+.audio-player-progress::-webkit-progress-value {
+  background: white;
+}
+
+.audio-player-progress::-webkit-progress-bar {
+  background: #ebc1b6;
+}
+
+.audio-player-progress::-moz-progress-bar {
+  background: white;
+}
+// sass-lint:disable no-vendor-prefixes
+
+.audio-player-time-text-right {
+  text-align: left;
+  padding-left: 12px;
+}
+
+.audio-player-playback-speed-container {
+  padding-left: 12px;
+}
+
+.audio-player-playback-speed-container button {
+  background-color: inherit;
+  border: 1px solid white;
+  border-radius: 4px;
+}
+
+.amplitude-playback-speed-10:before {
+  content: "1.0x";
+}
+
+.amplitude-playback-speed-15:before {
+  content: "1.5x";
+}
+
+.amplitude-playback-speed-20:before {
+  content: "2.0x";
+}
```

Comment:
small scss comment - when I'm styling something like this (something we're building as a real self-contained UI component) I like to scope all the styles specific to it within an outer selector (like `.audio-player-container-outer`) which will just make sure that all these rules don't accidentally affect anything outside of it.

of course I think we're unlikely to have elements elsewhere on the site that will be selected by `.amplitude-playback-speed-15:before` but still a good general rule to follow I think :smile:

Reply:
The latest commits to the branch should show everything nested nicely.